merlin/server/merlin_server.yaml

```diff
@@ -9,8 +9,8 @@ container:
     url: docker://redis
     # The config file
     config: redis.conf
-    # Subdirectory name to store configurations Default: merlin_server/
-    config_dir: merlin_server/
+    # Directory name to store configurations Default: ./merlin_server/
+    config_dir: ./merlin_server/
```

Comment:
Where will ./merlin_server/ be by default?

Reply:
Merlin server will still be used for user and container configuration but system configuration will now use app.yaml Before it used a file called merlin_server.yaml that was stored in ~/.merlin/server/.